ansible/playbooks/roles/repository/files/download-requirements/redhat-7/add-repositories.multiarch.sh

```diff
@@ -15,7 +15,7 @@ ELASTIC_6_REPO_CONF=$(cat <<'EOF'
 name=Elastic repository for 6.x packages
 baseurl=https://artifacts.elastic.co/packages/oss-6.x/yum
 gpgcheck=1
-gpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch
+gpgkey=https://artifacts.elastic.co/GPG-KEY-opensearch
```

Comment:
This will have to be added to the new download-requirements: [2926](https://github.com/epiphany-platform/epiphany/pull/2926)

Reply:
Outdated.